examples/imagenet/configs/default.py

```diff
@@ -36,11 +36,14 @@ def get_config():
 
   # As defined in the `models` module.
   config.model = 'ResNet50'
+  # We use 'imagenette' in the Colab.
```

Comment:
```suggestion
  # Allow the dataset to be configurable, since we fine-tune on 'imagenette' in the Colab.
```

Reply:
In the current Colab we don't fine-tune but rather train from scratch on imagenette, then load a pre-trained model from cloud and evaluate on iamagenette. I renamed the sections ("train from scratch", "load pre-trained") to make this a bit clearer, PTAL.

Maybe we should modify the imagenet example and gear it more towards transfer learning, WDYT?

If we want to do so, then maybe we should add some scripts for this as well and update the README accordingly - in a separate PR.

Reply:
But anyways, I don't think this comment is really useful. I'd rather say what form the string should have.